xAPI.md

```diff
@@ -510,6 +513,63 @@ Clarifications and additional examples including:
 - Addiitional explanitory text and diagrams
 [1.0.2...1.0.3](https://github.com/adlnet/xAPI-Spec/compare/1.0.2...1.0.3)
 
+
+<a name="Appendix1B"/>
+### Appendix B: cmi5 Example 
+
+The following example from the cmi5 Community of Practice demonstrates Extensions, "category", etc.
```

Comment:
I'm not keen on the use of "etc.". I propose the following text instead:

The following example illustrates a statement following the cmi5 Community of Practice. It demonstrates use of Extensions and "category" Context Activities.